backend/controllers/face_payment.go

```diff
@@ -44,7 +48,7 @@ func (ctrl *Controller) GetActiveAccountBySessionID(ctx *gin.Context) {
 			"error":      err,
 			"session_id": sessionId,
 			"data":       account,
```

Comment:
Ini kalo emang not found yaudah berarti ga perlu show `"data": account` karena isinya kosong